tests/bluechi_test/test.py

```diff
@@ -81,7 +82,7 @@ def setup(self) -> Tuple[bool, Tuple[BluechiControllerMachine, Dict[str, Bluechi
                                                       self.bluechi_controller_config)
             if self.run_with_valgrind:
                 ctrl_container.enable_valgrind()
-            ctrl_container.exec_run('systemctl start bluechi-controller')
+            SystemCtl(ctrl_container).start_unit("bluechi-controller")
```

Comment:
`ctrl_container` has an instance of `SystemCtl` as member:
```suggestion
            ctrl_container.systemctl.start_unit("bluechi-controller")
```

Reply:
Done